main/test/ca/uwaterloo/flix/library/TestGraph.flix

```diff
@@ -0,0 +1,116 @@
+namespace TestGraph {
+
+
+    def graph01(): Set[(Int32, Int32)] =
+        let _graphString =
+            " ┌─┐   ┌─┐ " ::
+            " │1├─→─┤2│ " ::
+            " └─┘   └─┘ " ::
+            Nil;
+        Set#{(1, 2)}
+
+    def graph02(): Set[(Int32, Int32)] =
+        let _graphString =
+            " ┌─┐   ┌─┐ " ::
+            " │1├─→─┤2│ " ::
+            " └─┘   └─┘ " ::
+            " ┌─┐   ┌─┐ " ::
+            " │3├─→─┤4│ " ::
+            " └─┘   └─┘ " ::
+            Nil;
+        Set#{(1, 2), (3, 4)}
+
+    def graph03(): Set[(Int32, Int32)] =
+        let _graphString =
+            " ┌─┐   ┌─┐ " ::
+            " │1├─→─┤2│ " ::
+            " └─┘   └┬┘ " ::
+            "        ↓  " ::
+            "       ┌┴┐ " ::
+            "       │3│ " ::
+            "       └─┘ " ::
+            Nil;
+        Set#{(1, 2), (2, 3)}
+
+    def graph04(): Set[(Int32, Int32)] =
+        let _graphString =
+            " ┌─┐   ┌─┐ " ::
+            " │1├─→─┤2│ " ::
+            " └─┘   └┬┘ " ::
+            "        ↓  " ::
+            " ┌─┐   ┌┴┐ " ::
+            " │4├─←─┤3│ " ::
+            " └─┘   └─┘ " ::
+            Nil;
+        Set#{(1, 2), (2, 3), (3, 4)}
+
+
```

Comment:
I think it would be nice to have one or two more graphs that are a bit bigger and have several larger components.

Reply:
I think they will come, when I have to do the strongly connected components and such :-)